tests/test_core.py

```diff
@@ -201,54 +207,58 @@ def test_insert_tables(engine_testaccount):
 
         # inserts data with the given id
         ins = users.insert()
-        conn.execute(ins, id=2, name="wendy", fullname="Wendy Williams")
+        with conn.begin():
+            conn.execute(ins, {"id": 2, "name": "wendy", "fullname": "Wendy Williams"})
 
         # verify the results
-        s = select([users])
+        s = select(users)
         results = conn.execute(s)
         assert len([row for row in results]) == 2, "number of rows from users table"
         results.close()
 
         # fetchone
-        s = select([users]).order_by("id")
+        s = select(users).order_by("id")
         results = conn.execute(s)
         row = results.fetchone()
         results.close()
```

Comment:
I know this is old code, but is closing the query results necessary?

Reply:
I'm not sure, some tests are really hard to read even without these, some are okay.
I was just wondering whether we should have them everywhere.